client/verta/verta/client.py

```diff
@@ -557,7 +557,7 @@ def set_experiment_run(self, name=None, desc=None, tags=None, attrs=None, id=Non
 
         return self._ctx.expt_run
 
-    def get_or_create_repository(self, name=None, workspace=None, id=None):
+    def get_or_create_repository(self, name=None, workspace=None, id=None, public_within_org=None):
```

Comment:
@evgeniilezhnin Could you also add `public_within_org` to the docstring's **Parameters** section? Should be able to copy&paste from `set_project()`.

Reply:
yes, added